Creates a new Streaks Card (#3338)
anuraghazra/github-readme-stats: readme.md

> ***

Reviewer: ```suggestion
```

# GitHub Streaks Card

> GitHub Streaks Card allows you to show your streak of your contributions on GitHub(like total contributions, weekly and longest streak on daily and weekly basis).

Reviewer: ```suggestion
The GitHub streaks card highlights your GitHub journey, summarizing total contributions, weekly activity, and your best daily and weekly streaks.
```

### Usage

Copy-paste this code into your readme and change the links.

Endpoint: `api/streak/?username=YOUR_USERNAME`

```md
![Streaks Card](https://github-readme-stats.vercel.app/api/streak/?username=YOUR_USERNAME)
```

### Hide individual elements from the streaks card

To hide individual elements form the card, use - `&hide=params`, where `params` = `totalContributions`, `weeklyLongest`, `weeklyCurrent`, `dailyLongest`, `dailyCurrent`

> 

Reviewer: ```suggestion

***
```
